docs/DeveloperGuide.adoc

```diff
@@ -729,27 +743,125 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |teacher |add new timeslot in timetable |I can plan my teaching schedule
+
+|`* * *` |teacher |add new location for a module in timetable |I can plan my teaching schedule
+
+|`* * *` |teacher |delete an existing module in a particular timeslot |I can make changes to the timeslot
+
+|`* * *` |teacher |delete an existing module in a particular location |I can make changes to the timeslot
+
+|`* * *` |teacher |update an existing module timeslot |I can change the current timeslot to an updated one
+
+|`* * *` |teacher |update an existing module location |I can change the current location to an updated one
+
+|`* * *` |teacher |list all the locations |I know all the locations I can choose from
+
+|`* * *` |teacher |view the help file  |I know all the available functions
+
+|`* * *` |teacher |view available slots at a specified location |I can determine if i should schedule my lecture there
+
+|`* * *` |teacher |login with my username/password |I can gain access to the product
```

Comment:
`I can gain access to the product` doesn't sound like a true benefit? How does gaining access via a password help? Try coming up with an appropriate benefit for the feature.

Reply:
noted with thanks, amended